client/bridge_client.go

```diff
@@ -172,6 +172,34 @@ func (ec *Client) BridgeGetSentChainTxsLimit(ctx context.Context) (uint64, error
 	return result, err
 }
 
+// BridgeRegisteredOperators can register an operator.
```

Comment:
```suggestion
// BridgeRegisteredOperators registers an operator.
```
How about remove all `can`  in comments?

Reply:
fixed. :)